src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/energy/EnergyNet.java

```diff
@@ -275,8 +272,8 @@ private void updateHologram(@Nonnull Block b, double supply, double demand) {
     private static EnergyNetComponent getComponent(@Nonnull Location l) {
         SlimefunItem item = BlockStorage.check(l);
 
-        if (item instanceof EnergyNetComponent) {
-            return ((EnergyNetComponent) item);
+        if (item instanceof EnergyNetComponent energyNetComponent) {
+            return energyNetComponent;
```

Comment:
We do not always have to be so literal about it.
```suggestion
        if (item instanceof EnergyNetComponent component) {
            return component;
```